pkg/dynamicrp/testhost/host.go

```diff
@@ -92,6 +97,9 @@ func Start(t *testing.T, opts ...TestHostOption) (*TestHost, *ucptesthost.TestHo
 	options, err := dynamicrp.NewOptions(context.Background(), config)
 	require.NoError(t, err)
 
+	// Prevent the default recipe drivers from being registered.
+	options.Recipes.Drivers = map[string]func(options *dynamicrp.Options) (driver.Driver, error){}
+
```

Comment:
Now we can load/use/mock the recipe engine in tests.

Reply:
This will be used in the next PR, where we're actually going to test a resource that uses recipes.